chcemvediet/templates/styleguide/typo/main.html

```diff
@@ -61,4 +61,14 @@ <h4>Heading 4</h4>
       <p>{% lorem 12 %}</p>
     </div>
   </div>
+  <div class="row">
+    <div class="col-md-4 chv-above-1">
+      <div class="chv-footnote">
```

Comment:
V styleguide to tiez presun medzi custom komponenty.

Reply:
:ok: